### PR TITLE
Address DEV warnings

### PR DESCRIPTION
**Pull request by bvaughn**
So far as I can tell from my spot testing, this does not result in any user-facing behavioral change.

### CODE REVIEW

File: src/devtools/client/debugger/src/components/App.tsx [hunk 1]
```diff
@@ -34,6 +34,11 @@ type DebuggerProps = PropsFromRedux & { wrapper: HTMLDivElement };
 class Debugger extends Component<DebuggerProps> {
   shortcuts = new KeyShortcuts({ window, target: this.props.wrapper });
 
+  static childContextTypes = {
```
Comment (markerikson) [inline src/devtools/client/debugger/src/components/App.tsx:37]: :hand: Just add `static` in front of the other `childContextTypes` field below, which is what I was _trying_ to do when I moved that inside the class.  (or move it back out and do `Debugger.childContextTypes = ` after the definition, which is how I was going to fix this myself.)

File: src/ui/actions/session.ts [hunk 2]
```diff
@@ -165,31 +166,51 @@ export function createSocket(
 
       const loadPoint = new URL(window.location.href).searchParams.get("point") || undefined;
 
+      const queuedActions: Array<Action | UIThunkAction> = [];
+      let flushTimeoutId: NodeJS.Timeout | null = null;
+
+      function flushQueuedActions() {
+        flushTimeoutId = null;
+
+        const actions = queuedActions.splice(0);
```
Comment (markerikson) [inline src/ui/actions/session.ts:175]: :question: I'm confused by the `.splice(0)`. What's that doing?

File: src/devtools/client/debugger/src/components/App.tsx (hunk 1 above)
Comment (bvaughn) [reply src/devtools/client/debugger/src/components/App.tsx:37]: Oh fuck. I ... missed the other one 😮‍💨 It's been a day

File: src/ui/actions/session.ts (hunk 2 above)
Comment (bvaughn) [reply src/ui/actions/session.ts:175]: Grabbing all of the current actions _and_ clearing out the queue so we can add more actions to it later.

```js
const actions = queuedActions.splice(0);

// actions now has all of the items queuedActions used to have
// queuedActions is now empty
```

Comment (markerikson) [reply src/ui/actions/session.ts:175]: Ah. I probably woulda just replaced the existing array or something :)

**Pull request merged**